index.js

```diff
@@ -590,7 +600,7 @@ converters.Identifier = (path, context) /*: K.Id*/ => {
   let kind = getIdentifierKind(path);
```

Comment:
TODO - split Identifier logic out into its own file and refactor/add comments to make this more useable